source/javascripts/controllers/_MainController.js.erb

```diff
@@ -686,13 +686,20 @@ import datadogRumCustomTiming from "../utils/datadogCustomRumTiming.ts";
 				})
 
 				if (requestService.mode === 'website') {
-					const guideMetadataReq = requestService.getCurrentUserMetadata('show_getting_started_guide');
-					const lastWFEDateMetadataReq = requestService.getCurrentUserMetadata('last_workflow_edited_date');
-					const tourReq = requestService.getProductTourShown();
-					Promise.all([guideMetadataReq, lastWFEDateMetadataReq, tourReq]).then(([guideMetadataResp,lastWFEDateMetadataResp, tourResp]) => {
-						viewModel.guidedOnboardingEnabledForUser = guideMetadataResp;
-						viewModel.lastWorkflowEditedDate = lastWFEDateMetadataResp;
-						viewModel.productTourShown = tourResp;
+					
+					launchDarklyService.initialize().then(() => {
+						const isProductTourEnabled = launchDarklyService.variation('enable-product-tours');
+						const guideMetadataReq = requestService.getCurrentUserMetadata('show_getting_started_guide');
+						const lastWFEDateMetadataReq = requestService.getCurrentUserMetadata('last_workflow_edited_date');
+						const tourReq = isProductTourEnabled ? requestService.getProductTourShown() : null;
+						
+						const requests = _.compact([guideMetadataReq, lastWFEDateMetadataReq, tourReq]);
+
+						Promise.all(requests).then(([guideMetadataResp,lastWFEDateMetadataResp, tourResp]) => {
+							viewModel.guidedOnboardingEnabledForUser = guideMetadataResp;
+							viewModel.lastWorkflowEditedDate = lastWFEDateMetadataResp;
+							viewModel.productTourShown = tourResp ? tourResp.tourShown : true;
```

Comment:
Oh good, you found it. :)

Reply:
@jakerobb please check out - https://bitrise.atlassian.net/browse/PLG-1500?focusedCommentId=104036